src/components/AccountModule/utils.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react'
+import { useTheme } from '@aragon/ui'
+
+import { web3Providers } from '../../environment'
+import { getNetworkByChainId } from '../../network-config'
+import { getWeb3 } from '../../web3-utils'
+
+export const getWeb3Instances = () => {
+  const walletWeb3 = getWeb3(web3Providers.wallet)
+  const web3 = getWeb3(web3Providers.default)
```

Comment:
We don't really need this; we can just use the hardcoded `network` for now.

Reply:
:+1:  https://github.com/aragon/aragon/pull/1137/commits/912202ced31cdf5733619cdfb6fc0c47d8cef6cf